src/test/scala/edu/ie3/simona/event/listener/RuntimeEventListenerSpec.scala

```diff
@@ -131,9 +136,9 @@ class RuntimeEventListenerSpec
           s"Simulation until ${calcTime(currentTick)} completed."
         ),
         (
-          Done(endTick, duration, 0, errorInSim = false),
+          Done(endTick, duration, errorInSim = false),
           Level.INFO,
-          s"Simulation completed with \u001b[0;32mSUCCESS (Failed PF: 0)\u001b[0;30m in time step ${calcTime(endTick)}. Total runtime: 3h : 0m : 5s "
+          s"Simulation completed with \u001b[0;32mSUCCESS (Failed PF: 2)\u001b[0;30m in time step ${calcTime(endTick)}. Total runtime: 3h : 0m : 5s "
```

Comment:
Is this really completed with SUCCESS if there are failing PFs. Perhaps we should discuss and resolve this in another thread, as it wasn't caused by your changes..

Reply:
There used to be the setting `simona.time.stopOnFailedPowerFlow`, which now just has a different name. So, we can configure SIMONA to keep running when power flows fail (thus it still can succeed), or we can stop with an error.

Reply:
Ok, just tested this to be sure. 
- If you `simona.powerflow.stopOnFailure` is set to `true`, the `GridAgent` fails if pf does not converge. (It does so in a kind of ugly way, but it still fails. #660 should deal with this).
- If it's set to `false`, we count failed power flows correctly and end with "SUCCESS".